package.json

```diff
@@ -60,6 +61,8 @@
     "minimist": "^1.1.1",
     "nyc": "^12.0.2",
     "standard": "^10.0.0",
+    "tap-mocha-reporter": "^3.0.7",
```

Comment:
``tap-mocha-reporter`` isn't needed as a dependency here, or is it?

Reply:
You're right, it's not needed.

Reply:
Removed in 39cc0d5